capstone/src/main/java/com/google/step/clients/ScoreClient.java

```diff
@@ -0,0 +1,163 @@
+package com.google.step.clients;
+
+import com.google.step.data.RestaurantPageViews;
+import com.google.step.data.RestaurantScore;
+import com.google.step.data.WeeklyPageView;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Collections;
+
+
+/**
+ * Will take care of calculating the score for each restaurant.
+ */
+public class ScoreClient {
+
+    private final float minimum = 10.0f;
+    private final MetricsClient metricsClient = new MetricsClient();
+
+    /**
+     * Only method to be called and calculates the scores of each restaurant.
```

Comment:
Nit: Consider removing "Only method to be called and". Users should be able to tell this by looking for all the public functions. Also this way if you add another public function you don't need to change this comment.

Reply:
Done!